tutor/src/components/task-plan/homework/loading-exercises-mixin.cjsx

```diff
@@ -12,7 +13,8 @@ LoadingExercisesMixin =
   componentWillUnmount: -> ExerciseStore.removeChangeListener(@exerciseLoadListenerOnUpdate)
 
   requestExerciseLoad: ->
-    ExerciseActions.loadForCourse(@props.courseId, @props.sectionIds, @props.ecosystemId)
+    ecosystemId = @props.ecosystemId or TaskPlanStore.getEcosystemId(@props.planId, @props.courseId)
```

Comment:
so, we will always want to pass the ecosystem id when loading exercises from now on??